app/views/backdate/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("backdate.edit.title") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_govspeak(t("backdate.edit.intro_paragraph")) %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% legend = capture do %>
+      <span class="govuk-heading-s govuk-!-margin-bottom-0">
+        <%= t("backdate.edit.date") %>
```

Comment:
Minor, but could we pass this into the translation instead of rendering it separately? I almost didn't spot it on the end ;-).

Reply:
As discussed offline, moved the legend variable up to the top.